scalyr_agent/copying_manager.py

```diff
@@ -1026,6 +1026,18 @@ def generate_status(self, warn_on_rate_limit=False):
             for entry in self.__log_matchers:
                 result.log_matchers.append(entry.generate_status())
 
+            if self.__config.enable_health_check:
+                result.health_check_result = "Good"
+                if (
+                    time.time()
+                    > self.__last_attempt_time
+                    + self.__config.healthy_max_time_since_last_copy_attempt
+                ):
+                    result.health_check_result = (
```

Comment:
Can you add in a unit test that tests this method?  In particular, verify it returns a bad message if the last copy attempt has been exceeded.